.vitepress/sidebar.mts

```diff
@@ -9,20 +9,44 @@ export const sidebar: DefaultTheme.SidebarItem[] = [{
 			link: 'https://docs.meower.org/api/rest.html',
 		},
 		{
-			text: 'Cloudlink',
+			text: 'Cloudlink (websocket)',
 			collapsed: true,
 			items: [
 				{
 					text: 'Introduction',
 					link: '/api/cloudlink/intro',
 				},
 				{
-					text: 'authpswd command',
-					link: '/api/cloudlink/commands/authpswd',
+					text: 'Commands',
```

Comment:
The only command that still exists with ?v=1 is `ping` iirc - does there need to be a separate section for that?

Reply:
good point, let me change that.